cdap-ui/app/cdap/components/CaskWizards/PublishPipelineUsecase/index.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {PropTypes} from 'react';
+import PublishPipelineWizard from 'components/CaskWizards/PublishPipeline';
+
+export default function PublishPipelineUsecaseWizard({input, onClose, isOpen}) {
+  return (
```

Comment:
This should have been the place where we should have handled call to actions specific for PublishPipeline Wizard for market (use case). This wrapper now seems pointless with just setting `isUseCase` to true right?